test/e2e/storage/testsuites/subpath.go

```diff
@@ -587,8 +587,6 @@ func testPodFailSubpathError(f *framework.Framework, pod *v1.Pod, errorMsg strin
 	defer func() {
 		framework.DeletePodWithWait(f, f.ClientSet, pod)
 	}()
-	err = framework.WaitForPodRunningInNamespace(f.ClientSet, pod)
-	Expect(err).To(HaveOccurred(), "while waiting for pod to be running")
```

Comment:
Might we need to increase the timeout waiting for the event below, now that we're not waiting for the pod?

Reply:
@justinsb 

Thank you for pointing it out. 

Previous code wait for the pod not become running for 5 minutes(```PodStartTimeout```) and then wait for the event for 2 minutes(```PodEventTimeout```). And current code only wait for the event for 2 minutes, so the timeout is reduced much less than before. 
However, in my environment, these entire tests complete around 1 minutes including setup/teardown, so I guess that it would be enough for waiting events for these tests. And ```PodEventTimeout``` will match to the context for these tests. We will be able to tune the time once we find the tests flake due to the timeout.

Reply:
SGTM - thanks @mkimuram